frontend/constants/serviceTemplates.ts

```diff
@@ -6,7 +6,7 @@ import { StakingProgramId } from '@/enums/StakingProgram';
 export const SERVICE_TEMPLATES: ServiceTemplate[] = [
   {
     name: 'Trader Agent',
-    hash: 'bafybeidicxsruh3r4a2xarawzan6ocwyvpn3ofv42po5kxf7x6ck7kn22u',
+    hash: 'bafybeic26ovbgnb3l7dpgc33vy55ucv2ou7i7drkgcvohen72nhcso6iny',
```

Comment:
Is this trader_pearl departing from the same branch as the current trader on quickstart (v18.7)?.

Reply:
https://github.com/valory-xyz/trader-quickstart/blob/main/run_service.sh#L705